index.html

```diff
@@ -328,6 +334,25 @@ <h3>Subtitle and caption rendering synchronization</h3>
           and subtitle changes occur at the same time. Many subtitles therefore
           start on the first frame of the shot and end on the last frame."
         </p>
+        <p>
+         The NorDig technical specifications for DVB receivers for the Nordic
+         and Irish markets [[NORDIG]], section 7.3.1, mandates that receivers
+         support TTML in MPEG-2 Transport Streams. The presentation timing
+         precision for subtitles is specified as being within 2 frames.
+        </p>
+        <p>
+          Another important use case is maintaining synchronization of subtitles
+          during program content with fast dialog. The BBC Subtitle Guidelines,
+          section 5.1 says: "Impaired viewers make use of visual cues from the
```

Comment:
I think there's a way in html/respec of styling a block quote - consider doing that here?

Reply:
Thanks, I've added blockquotes for this now.